website/blog/2020-06-27-lithium.md

```diff
@@ -0,0 +1,503 @@
+  </tr>
+  <tr>
+    <td>Closed issues</td>
+    <td align="center">12</td>
+  </tr>
+  <tr>
+    <td>New features</td>
+    <td align="center">3</td>
+  </tr>
+</tbody>
+</table>
+
+For full details: https://github.com/scalameta/metals/milestone/24?closed=1
+
+Metals is a language server for Scala that works with VS Code, Vim, Emacs,
+Sublime Text, Atom and Eclipse. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+with the help from [Lunatech](https://lunatech.com) along with contributors from
+the community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/), and
+give Metals a try!
+
+- Ammonite support
+- new project provider
+- worksheet dependency support
+- Scala 2.13.3, 0.24.0, 0.25.0-RC1 support
+
+## Ammonite support
+
+Thanks to a great work by [alexarchambault](https://github.com/alexarchambault)
+we know support [Ammonite](https://ammonite.io) script files. Ammonite enables
+to create scripts that you can run easily from the command line, without the
+overhead of setting build tool while still allowing you to download and use
+dependencies.
+
+The support includes most of the LSP features such as diagnostics, completions,
+references, go to definition, but does not include running or debugging the
+scripts. This new feature works by starting an additional Ammonite BSP server
+that compiles the scripts and provides additional information about them such as
+scala version or information about the added dependencies. You can change the
+default JVM parameters for the server such as `-Xmx`, which will limit the used
+memory in Metals settings.
+
+Each of the scripts needs to be first imported into Ammonite BSP server, which
+can be done manually by clicking on the pop up whenever new script is opened or
+selecting automatic import at the start, which will import any open scripts.
+
+It's important to note that this is an early approach to the support, which
+might still require some improvements both on the Metals and Ammonite side. You
+learn more about the new feature or comment the in the
+[blogpost](https://contributors.scala-lang.org/t/ammonite-support-in-metals/4306)
+on the Scala contributors forum.
+
+Most notable features of Ammonite scripts are:
+
+- importing other scripts, Metals provide completions to select existing files
+  based on the filesystem information
+
+```scala
+//main.sc
+import $file.hello-world
+val greeting = hello-world.greeting
+
+//hello-world.sc
+val greeting = "Hello world!"
+```
+
+- importing dependecies
+
+```scala
+import $ivy.`com.lihaoyi::scalatags:0.7.0`
+import scalatags.Text.all._
+
+val rendered = div("Moo").render
+```
+
+## Worksheet dependency support
+
+In addition to Ammonite support, which will be useful to anyone interested in
+Scala scripting, we've also improved the support for Metals Scala worksheets.
+Thanks to the efforts of [olafurpg](https://github.com/olafurpg), we now are
```

Comment:
```suggestion
Thanks to the efforts of [olafurpg](https://github.com/olafurpg) in [mdoc](https://scalameta.org/mdoc/), which powers worksheets, users are now
```

Reply:
What do you think about mentioning mdoc here? I think it's a good idea to give credit where credit is due, but also I've found that a lot of people are unaware that mdoc is how the worksheets are actually working.

Reply:
Sure, good idea! I haven't thought of it.